app/views/goodsProfile/GoodsRecordsView.scala.html

```diff
@@ -69,11 +69,7 @@
                     totalRecords match {
                     case 0  => messages("goodsRecords.title.noResults")
                     case 1  => messages("goodsRecords.title.singleRecord")
-                    case _  =>
-                        pagination.items match {
-                            case Some(items) if items.size > 1 => messages("goodsRecords.title.withPageFilter", totalRecords, page.toString,page, items.size.toString)
-                            case _ => messages("goodsRecords.title")
-                        }
+                    case _  => messages("goodsRecords.title.withPageFilter", totalRecords.toString, page.toString, pagination.items.map(_.size).getOrElse(0).toString)
```

Comment:
would this not need to be getOrElse 1? Otherwise if it doesn't exist it'd show as page 1 of 0?